src/widgets.js

```diff
@@ -267,6 +267,12 @@ define([
             return widget.input.val();
         };
 
+        mug.on("property-changed", function (e) {
```

Comment:
Hmm, another dependency on `property-changed`, which is ["legacy"](https://github.com/dimagi/Vellum/blob/7f8341908a1a1dad88962285588df0075c95d2c1/src/form.js#L890-L892). I don't really have a better solution, so not saying it should be done differently, just pointing it out. I think I did a similar thing elsewhere in this module.

Musing here: wonder if we should remove that "legacy" comment and formally endorse usage of this event, especially now that we have a mechanism for removing event listeners on `teardown-mug-properties`? Also wondering if that was the reason why Mike wanted to avoid more dependencies on `property-changed` or if there was some other reason?